LunaDll/GlobalFuncs.h

```diff
@@ -277,3 +278,6 @@ constexpr std::uint32_t DoubleMostSignificantDWord(double d) {
 #endif
 
 #endif
+
+extern std::mutex g_editorIPCMutex;
```

Comment:
I was about to merge this.... but probably one last issue:
Why is `g_editorIPCMutex` defined and implemented in GlobalFuncs.h+cpp whereas `gEditorPlacedItem` is defined and implemented in Globals.h+cpp? These are variables that are closely associated, and every place that needs to access one needs to access the other by definition.

Come to think of it, there could also make an argument they should be put into a struct (or even a class that prevents non-thread-safe access), but that's not overly important. They should at least be alongside eachother in the same files rather than randomly split apart for no reason.